### ISSUE TITLE
Add "how to" guides for each provider

### ISSUE THREAD

**Issue by ianaya89** [checkly/jamstack-deploy#11]

**Comment by kokamkarsahil** [comment]
Hello, I would like to work on it. Can we discuss it?
Thank You!

**Comment by ianaya89** [comment]
Hi @kokamkarsahil, thanks for your interest.

What I have in mind is that each provider has a descriptive page where you can easily read how to start using it and how to do your first deployment.
I would also like to include a list of features that each provider has (SSL, custom domains, cli, UI, etc).
We can start working on different MD files that we can keep in the repo as docs and maybe in the next iteration put that information in the web app (design needed)

What do you think? Is something you would like to do?

**Comment by kokamkarsahil** [comment]
Thanks for reply!

> What I have in mind is that each provider has a descriptive page where you can easily read how to start using it and how to do your first deployment.

Sure, I am bit confused in layout of the page. Like how the things will be nested.

> I would also like to include a list of features that each provider has (SSL, custom domains, cli, UI, etc).

Got it. Like we can difference their free and paid offerings or just include whether it's supported without mentioning free or paid tier?

> We can start working on different MD files that we can keep in the repo as docs and maybe in the next iteration put that information in the web app (design needed)

Sure! I won't mind using markdown.

> What do you think? Is this something you would like to do?

I was planning on using static doc generators like [kit docs](https://kit-docs.svelteness.dev), [docusarus](https://docusaurus.io), [vuepress](https://vuepress.vuejs.org) or any other generators. But now I don't know how the web app will be styled, like how the layout will be?

Thank You! Would like to get clarified with the above stuff, then I will create a demo page with the above things in mind.

**Comment by kokamkarsahil** [comment]
Any updates @ianaya89 ?

**Comment by kokamkarsahil** [comment]
Yup, we can talk about it in next weak. And sorry for pining didn't knew about the new release. I have reached you via email
My email: Sahil Kokamkar <sahil@kokamkar.com>

**Comment by kokamkarsahil** [comment]
Hi @ianaya89,
I created a demo https://jamstack-docs.vercel.app
And a full doc for AWS-amplify: https://jamstack-docs.vercel.app/docs/providers/aws-amplify
It's still W.I.P.
Let me know if there are any issue or improvements.
BTW didn't got any response through email.

**Comment by ianaya89** [comment]
Hi @kokamkarsahil, so sorry for the silence I am on parental leave right now. Thanks a lot for the contribution, will take a deeper look this week and get back to you with feedback 💯

**Comment by kokamkarsahil** [comment]
> Hi @kokamkarsahil, so sorry for the silence I am on parental leave right now.

Sorry for pinging, didn't know you were on leave. 

> Thanks a lot for the contribution, will take a deeper look this week and get back to you with feedback :100: 

Sure no problem, will be waiting for your feedback :+1:  And have a great day!

**Comment by ianaya89** [comment]
Hi @kokamkarsahil, no problem at all. I just reviewed what you sent and looks fantastic. We can later discuss in more detail the sections and topics, but this is precisely what I have in mind.

Which tech stack are using for this? I would like to keep it in this same repo and use our existing app/repo. Do you think we can make it work?

Thanks!

**Comment by kokamkarsahil** [comment]
> Hi @kokamkarsahil, no problem at all. I just reviewed what you sent and looks fantastic. We can later discuss in more detail the sections and topics, but this is precisely what I have in mind.

Yeah sure!

> Which tech stack are using for this?

I have used [kit docs](https://github.com/svelteness/kit-docs) which is based on svelte kit. And using Playwright for E2E testing + eslint. You also checkout the [source code](https://gitlab.com/kokamkarsahil/jamstack-docs).

> I would like to keep it in this same repo and use our existing app/repo. Do you think we can make it work?

Yes we can with the help of `vercel.json`. In case if the app size increases we can use something like [TurboRepo](https://turborepo.org/).

> Thanks!

Happy To Contribute! :hugs:

**Comment by ianaya89** [comment]
Looks, great. What do you think about using Vuepress (or nuxt) for this? I would like to avoid having different tech stacks for the app and the docs. I think will be better for other contributors in the future, improve code re-using and make the project cleaner

**Comment by kokamkarsahil** [comment]
> @kokamkarsahil  Looks great. What do you think about using Vuepress (or nuxt) for this?

Yes it will be good and we can easily switch to it without any problem, but what do you think about  [vitepress] (https://vitepress.vuejs.org)?

> I would like to avoid having different tech stacks for the app and the docs. I think will be better for other contributors in the future, improve code re-using and make the project cleaner

Yeah that makes sense. And how about using [MDX](https://mdxjs.com) instead of Markdown?

**Comment by ianaya89** [comment]
@kokamkarsahil I am ok with both.  
If vitepress is mature enough, let's go. MDX is also a great choice!

**Comment by kokamkarsahil** [comment]
> @kokamkarsahil I am ok with both.  
> If vitepress is mature enough, let's go. MDX is also a great choice!

Unfortunately, both [vuepress] ( https://vuepress.vuejs.org/guide/markdown.html) and [vitepress](https://vitepress.vuejs.org/guide/markdown.html) doesn't have support for steps using in the docs.

i.e The steps and number style used in markdown [check the numbers and steps used here](https://jamstack-docs.vercel.app/docs/providers/aws-amplify#getting-started).
